src/commands/stashDelete.ts

```diff
@@ -19,8 +20,16 @@ export class StashDeleteCommand extends Command {
         super(Commands.StashDelete);
     }
 
-    async execute(args: StashDeleteCommandArgs = { confirm: true }) {
+    async execute(args: StashDeleteCommandArgs | StashCommitNode = { confirm: true }) {
```

Comment:
@rebornix Can we use the new `run` method like here: https://github.com/eamodio/vscode-gitlens/blob/feature-explorer/src/commands/diffWithPrevious.ts#L25

That way the dealing with the node and refreshing it could be specific for that context?